feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/detail/BalanceDetailFragment.kt

```diff
@@ -137,6 +143,8 @@ class BalanceDetailFragment : BaseFragment<BalanceDetailViewModel>() {
 
         viewModel.sendEnabled.observe(balanceDetaiActions.send::setEnabled)
 
+        viewModel.sendEnabled.observe(balanceDetaiActions.send::setEnabled)
```

Comment:
Seems to be a dublicated line
![image](https://github.com/novasamatech/nova-wallet-android/assets/70131744/eec89618-eedb-4c41-808f-fe72329c2ccb)